PyISY/Nodes/group.py

```diff
@@ -106,3 +107,12 @@ def on(self):
         else:
             self.parent.parent.log.info('ISY turned on scene: ' + self._id)
             return True
+
+    def _get_notes(self):
+        self._notes = self.parent.parent.conn.getNodeNotes(self._id)
```

Comment:
I'm not convinced this method needs to exist. This line can be placed in the spoken property.